### DISCUSSION TITLE
alt gr key not working after update to 0.7.2-1

### DISCUSSION THREAD

**Discussion by sandrock** [qwerty-fr/qwerty-fr#73] [Q&A] [answered]
Bonjour,

Après avoir mis à jour mon système (Manjaro, KDE), la touche Alt Gr n'a plus aucun effet.   
After a system update (Manjaro, KDE), the right Alt has no effect any more.   

J'ai alors configuré 2 claviers dans KDE : `us_querty-fr` et `fr` afin de pouvoir écrire en français au besoin. Voici ce que dit `setxkbmap -print` :    
I temporarily configured two layouts in KDE: `us_querty-fr` et `fr` to allow me writing in french. Here is the `setxkbmap -print` output:    

```
xkb_keymap {
	xkb_keycodes  { include "evdev+aliases(qwerty)"};
	xkb_types     { include "complete"	};
	xkb_compat    { include "complete"	};
	xkb_symbols   { include "pc+us_qwerty-fr+fr:2+inet(evdev)+capslock(none)+terminate(ctrl_alt_bksp)+group(shifts_toggle):1+group(shifts_toggle):2"	};
	xkb_geometry  { include "pc(pc104)"	};
};
```

Alt Gr ne fonctionne d'ailleurs pas avec le layout `fr`...   
The key does not work with any layout. 

Quelqu'un aurait une idéé ?

**Comment by devnoname120** [comment]
Did you try disabling the Compose key in your settings?
https://userbase.kde.org/Tutorials/ComposeKey

**Comment by sandrock** [comment]
There is nothing checked in the "Position of the compose key" in the KDE settings. My keyboard has a right Alt key so I believe I do not need to change anything there. 

Some more info:

- keyboard model: [cherry stream](https://www.cherryamericas.com/stream-keyboard?___store=cherry_us_en) with layout "eu usa with euro symbol"
- keyboard model selected in KDE: generic 104 keys PC
- I was previously using openbox as desktop env. Maybe an old config creates interference? (I did not find anything)
- /etc/X11/xorg.conf.d/00-keyboard.conf:
  ```
  Section "InputClass"
        Identifier "system-keyboard"
        MatchIsKeyboard "on"
        Option "XkbLayout" "us"
        Option "XkbModel" "pc105+inet"
        Option "XkbOptions" "terminate:ctrl_alt_bksp"
  EndSection
  ```
- this app `rssguard` crashes at startup because of the keyboard. 
  ```
  $ rssguard
  time="     0.000" type="debug" -> core: Starting RSS Guard 4.6.3.
  time="     0.000" type="debug" -> core: Current UTC date/time is 2024-03-10T18:40:38Z.
  time="     0.071" type="debug" -> core: Initializing settings in '/home/sandrock/.config/RSS Guard 4/config/config.ini' (non-portable way).
  time="     0.071" type="debug" -> core: Starting to load active localization. Desired localization is 'en_GB'.
  time="     0.071" type="debug" -> core: Application localization 'en_GB' loaded successfully, specifically sublocalization 'en_GB' was loaded.
  time="     0.071" type="warning" -> core: Qt localization 'en_GB' WAS NOT loaded successfully.
  xkbcommon: ERROR: Unrecognized RMLVO variant "fr" was ignored
  xkbcommon: ERROR: Unrecognized RMLVO variant "fr" was ignored
  xkbcommon: ERROR: [XKB-338] Couldn't find file "symbols/us_qwerty" in include paths
  xkbcommon: ERROR: [XKB-338] 1 include paths searched:
  xkbcommon: ERROR: [XKB-338] 	/usr/share/X11/xkb
  xkbcommon: ERROR: [XKB-769] Abandoning symbols file "(unnamed)"
  xkbcommon: ERROR: Failed to compile xkb_symbols
  xkbcommon: ERROR: Failed to compile keymap
  [1232486:1232508:0310/194038.470583:FATAL:xkb_keyboard_layout_engine.cc(651)] Keymap file failed to load: us_qwerty-fr,fr
[1]    1232486 trace trap (core dumped)  rssguard
  ```

**Comment by sandrock** [comment, marked as answer]
Ho. Suddenly, it works. No reboot or config change. I came back to the PC and it just worked. Closing.
